Dockerfile

```diff
@@ -20,6 +20,8 @@ FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
 WORKDIR /root/
 COPY --from=builder /go/src/github.com/openshift/cloud-credential-operator/manager .
 ADD manifests/ /manifests
+# Update perms so we can copy updated CA if needed
+RUN chmod -R g+w /etc/pki/ca-trust/extracted/pem/
```

Comment:
@joelddiaz Do we need to give write permission to files under /etc/pki/ca-trust/extracted/pem/ directory? If we just need copy updated CA, is it enough to only give write permission to directory?
I just have a doubt. If it doesn't influences, please ignore my question.

Reply:
If there is an existing file with RO permissions, then only changing the directory will still cause a failure.